recipes/sophios/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "sophios" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PolusAI/workflow-inference-compiler/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5c71e369346ba58ccace86897ff8d7cc4dadddb23380af51217a1ca03ecbe41c
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
Looks like this one can be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.